actionSHARK/mongo.py

```diff
@@ -160,8 +159,10 @@ def __init__(
             db_authentication_database,
             db_ssl_enabled,
         )
-
-        self.__conn = connect(db_database, host=self.__conn_uri)
+        try:
+            self.__conn = connect(db_database, host=self.__conn_uri)
+        except ConnectionFailure:
```

Comment:
You can also log the trace:
except ConnectionFailure:
    logger.exception('error connecting to mongodb')